packages/experimental/kai/src/app/pages/SpacePage.tsx

```diff
@@ -37,8 +40,8 @@ const ViewContainer: FC<{ view: string }> = ({ view }) => {
         content: { children: <Sidebar /> }
       }}
     >
-      <ViewSelector />
-      <div className='pbs-[84px] flex h-screen bg-white'>
+      {views.length > 1 && <ViewSelector />}
+      <div className={mx(views.length > 1 ? 'pbs-[84px]' : 'pbs-[48px]', 'flex h-screen bg-white')}>
```

Comment:
84 is the height of the AppBar?

Reply:
48 is the height of the AppBar and 84 is the height of AppBar+ViewSelector